### PR TITLE
feat: water mask for multi edition

### PR DESCRIPTION
**Pull request by Clauszy**
Display watermark according to authorized version.

Log: 桌面水印增加授权信息

Task: https://pms.uniontech.com/task-view-159413.html
Influence: 桌面水印

### CODE REVIEW

File: src/dde-desktop/presenter/deepinlicensehelper.cpp [hunk 1]
```diff
@@ -77,7 +77,7 @@ void DeepinLicenseHelper::initFinshed(void *interface)
     Q_ASSERT(!m_licenseInterface);
     m_licenseInterface = static_cast<ComDeepinLicenseInterface *>(interface);
     connect(m_licenseInterface, &ComDeepinLicenseInterface::LicenseStateChange,
-                     this, &DeepinLicenseHelper::licenseStateChanged);
+                     this, &DeepinLicenseHelper::requetLicenseState);
```
Comment (itsXuSt) [inline src/dde-desktop/presenter/deepinlicensehelper.cpp]: request?

**Comment by itsXuSt** [comment]
[是否检查兼容性] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Review by itsXuSt — DISMISSED** [review]
[是否检查兼容性] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Review by Johnson-zs — APPROVED** [review]

**Pull request closed without merging**